client/src/main/js/controller/LocalController.js

```diff
@@ -1,39 +1,83 @@
 (() => {
     'use strict';
 
-    angular.module('localModulo').controller('LocalController', ['$state', '$stateParams', 'APP_STATES', 'Local', 'local', function ($state, $stateParams, APP_STATES, Local, local) {
-
+    angular.module('localModulo').controller('LocalController', ['$state', '$stateParams', 'APP_STATES', 'Local', 'local', 'LocaisService', 'ModalService', 'ToastService', function ($state, $stateParams, APP_STATES, Local, local, LocaisService, ModalService, ToastService) {
         const self = this;
 
         this.isAdmin = () => true;
 
         this.edicao = this.isAdmin();
         this.local = local;
 
+        /**
+         * Redireciona para tela de detalhes do local, após o mesmo ser clicado.
+         *
+         * @param local Local clicado.
+         */
         this.onChangeLocal = function (local) {
             if (podeRedirecionarLocal(local)) {
                 $state.go(APP_STATES.LOCAL_ID_INFO.nome, {idLocal: local._id});
             }
         };
 
+        /**
+         * Checa se o local clicado é válido para ser exibido.
```

Comment:
adiciona qual checagem que faz na documentação.

...true se local for definido e não tiver um local do controller ou...